setup_dependencies.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 """
-Manage dependancies by declaring systems here.
+Manage dependencies by declaring systems here.
 A system can depend on packages or other systems.
-If a package ends with a + include all subpackages.
+If a package ends with a '+', then all the subpackages are included as well.
```

Comment:
Here, again, it is not obvious why and how `+` will enforce sub-package inclusion. Is it done by a specific step/script? If so, where does it comes from? I looked at content of the file and can't find any indication how plus sign is converted into set of action described in the line above. I suggest to provide proper documentation to explain how it is done and which script/function will take care of collecting and including sub-package info.